registry/application/handlers/service_handlers.py

```diff
@@ -88,6 +88,27 @@ def save_service(event, context):
         {"status": "success", "data": response, "error": {}}, cors_enabled=True
     )
 
+@exception_handler(SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID, logger=logger)
+@secured(action=Action.CREATE, org_uuid_path=("pathParameters", "org_uuid"),
+         username_path=("requestContext", "authorizer", "claims", "email"))
+def save_service_attributes(event, context):
+    logger.info(f"Event for save service {event}")
+    username = event["requestContext"]["authorizer"]["claims"]["email"]
+    path_parameters = event["pathParameters"]
+    payload = json.loads(event["body"])
+    if not path_parameters.get("org_uuid", "") and not path_parameters.get("service_uuid", ""):
+        raise BadRequestException()
+    org_uuid = path_parameters["org_uuid"]
+    service_uuid = path_parameters["service_uuid"]
+    response = ServicePublisherService(username, org_uuid, service_uuid).save_service_attributes(payload)
```

Comment:
Can response never have an error?

Reply:
it can ave our exception handler annotation will take care of it.